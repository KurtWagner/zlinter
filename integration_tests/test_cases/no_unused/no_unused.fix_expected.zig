export const banana = @import("std");
pub const strawberry = @import("std");

const Tomato = struct {};
const Milk = struct {};

pub fn main() !void {
    const cookie = @import("std").ascii;
    _ = cookie;

    const tomato: Tomato = .{};
    tomato.* = undefined;

    const milk = Milk{};
    milk.* = undefined;
}

// zlinter-disable-next-line
const unused_but_disabled = 0;
const also_unused_disabled = 0; // zlinter-disable-current-line

// zlinter-disable-next-line no_unused
const also_explicitly_disabled = 0;

// zlinter-disable-next-line different_rule

// Comment
