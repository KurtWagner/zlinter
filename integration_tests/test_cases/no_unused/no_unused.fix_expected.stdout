5 fixes applied to: .zig-cache/tmp/no_unused.input.zig
Fixed 5 issues in 1 files!
3 issues disabled by comments.
19 fix passes remaining
0 fixes applied to: .zig-cache/tmp/no_unused.input.zig
Fixed 0 issues in 1 files!
3 issues disabled by comments.
Total of 5 issues fixed
